docs/kb/compaction.rst

```diff
@@ -188,7 +188,31 @@ Which Strategy is best to use
 -----------------------------
 Use the table in :ref:`Which strategy is best <which-strategy-is-best>` to determine the right strategy for your needs.
 
+.. _compaction-off-strategy-compaction:
 
+Off-strategy Compaction
+----------------------------
+
+Off-strategy compaction is a default mechanism that improves the efficiency of node operations, 
```

Comment:
```suggestion
Off-strategy compaction is an internal only compaction strategy used by ScyllaDB to improve the efficiency of node operations, 
```

Reply:
@raphaelsc please comment

Reply:
Anna is right. Off-strategy is an concept in a higher level than compaction strategy, that can apply to any of them